sdf/mandelbulbSDF.glsl

```diff
@@ -0,0 +1,39 @@
+#include "../space/spherical.glsl"
+
+/*
+author: Kathy McGuiness
+description: Returns the mandelbulb SDF
+use: mandelbulbSDF(<vec2> st)
+*/
+
+#ifndef FNC_MANDELBULBSDF
+#define FNC_MANDELBULBSDF
+vec2 mandelbulbSDF( in vec3 st ) 
+{
+  vec3 zeta = st;
```

Comment:
4 spaces as tabs

Reply:
OK, will fix.  VSC wasn't automatically formatting for me.  Do you have an extension you use?

Reply:
I don't : ( sorry.
I just set the tabs as 4 spaces.